### PR TITLE
change TiCDC to changefeed and label sink to kafka as beta

### PR DESCRIPTION
**Pull request by benmaoer**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [x] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: TOC-tidb-cloud.md [hunk 1]
```diff
@@ -137,7 +137,7 @@
   - [Changefeed Overview](/tidb-cloud/changefeed-overview.md)
   - [To MySQL Sink](/tidb-cloud/changefeed-sink-to-mysql.md)
   - [To Kafka Sink](/tidb-cloud/changefeed-sink-to-apache-kafka.md)
-  - [To TiDB Cloud Sink](/tidb-cloud/changefeed-sink-to-tidb-cloud.md)
+  - [To TiDB Cloud Sink (beta)](/tidb-cloud/changefeed-sink-to-tidb-cloud.md)
```
Comment (Oreoxmt) [inline TOC-tidb-cloud.md:140]:
```suggestion
  - [To TiDB Cloud Sink (Beta)](/tidb-cloud/changefeed-sink-to-tidb-cloud.md)
```

Please also update L2 and L6 to "Sink to TiDB Cloud (Beta)"

https://github.com/pingcap/docs/blob/d01a93dd22e64a74f305f62524dbab5f6fda70c3/tidb-cloud/changefeed-sink-to-tidb-cloud.md?plain=1#L1-L6

**Review by Oreoxmt — APPROVED** [review]
LGTM

**Review by hfxsd — APPROVED** [review]

**Review by benmaoer — COMMENTED** [review]
LGTM

**Review by grovecai — APPROVED** [review]

**Comment by grovecai** [comment]
LGTM

**Pull request merged**